### ISSUE TITLE
Refer to RFC 3987 instead of RFC 3986

### ISSUE THREAD

**Issue by fsasaki** [w3c/i18n-activity#934]
**This is a tracker issue.** Only discuss things here if they are i18n group internal meta-discussions about the issue. **Contribute to the actual discussion at the following link:**

§ https://github.com/w3c/miniapp-addressing/issues/1

**Comment by aphillips** [comment]
It's been a while since we visited this topic. Do we need to consider WHATWG's URL spec again?

**Comment by fsasaki** [comment]
Maybe I am not up to date with this topic. Years ago our group always pushed for references to RFC 3987, as a replacement for references to RFC 3986. Has this changed?

**Comment by aphillips** [comment]
No, it hasn't. But IRI remains a little controversial. That WHATWG doc was supposed to resolve that. In any case, my comment was basically asking "Has this changed?" :-)

**Comment by fsasaki** [comment]
OK, I see. Looking at https://url.spec.whatwg.org/ , it seems that asking for an RFC 3987 reference and the term "IRI" to be used is not likely to be welcomed.

**Comment by plehegar** [comment]
Issue got transferred to a new repo.
